### DISCUSSION TITLE
Uncaught ReferenceError: Cannot access 'tamu' before initialization

### DISCUSSION THREAD

**Discussion by adieska** [dewanakl/undangan#13] [Q&A]
Bang tolong bantu Bang, entah kenapa muncul masalah ini:

Uncaught ReferenceError: Cannot access 'tamu' before initialization
    at progress (app.js:23:13)
    at app.js:31:13
    at NodeList.forEach (<anonymous>)
    at app.js:28:12
    at app.js:39:3

Pertama kali kita kirim link ke tamu, dibuka di browser ga ada masalah, cuma kalo browser udah di refresh, undangannya ga bisa terbuka lagi. Jadinya hanya mentok di halaman awal, loading 100%.

**Comment by dewanakl** [comment]
Sebelumnya sudah download yang terbaru mas?

Pada tanggal Rab, 18 Okt 2023 16.28, adieska ***@***.***>
menulis:

**Comment by adieska** [comment]
Sudah Bang... Kira2 masalahnya dimana ya Bang?

**Reply by dewanakl** [reply]
Sudah saya update mas di githubnya, silahkan coba lagi

**Reply by adieska** [reply]
Baik Bang Dewana, terima kasih banyak atas responnya... Saya coba dulu Bang...  🙏
